README.md

```diff
@@ -48,38 +53,15 @@ If running in codespaces, this should already be done.
 
 ### AWS Setup
 
-The first thing you'll need to do is configure your aws environment with:
-
-```bash
-aws configure
-```
-
-Enter in your AWS Access Key ID and AWS Secret Access Key, which can be obtained by setting up a user account in the AWS console. For region, set it to the AWS region you'd like to set up your SDS. For IMAP, we're using "us-west-2"
-
-If you have multiple AWS access/secret key pairs locally, you can add the configuration to `~/.aws/config`. 
-
-```
-[imap]
-region=us-west-2
-aws_access_key_id=<Access Key>
-aws_secret_access_key=<Secret Key>
-```
-
-Then, you can set the profile used by cdk by setting the `AWS_PROFILE` environment variable to the profile name (in this case, imap):
-
-```
-export AWS_PROFILE=imap
-```
+[AWS Setup page](docs/source/cdk/aws-setup.rst)
 
 You may also need to set the `CDK_DEFAULT_ACCOUNT` environment variable. 
 
 **NOTE**-- For new AWS users, you'll need to make certain the AWS Cloud Development Kit is installed: 
 
-Ensure you have installed nodejs newer than version 14.
-<https://nodejs.org/en/download/>
 
 ```bash
-nvm use
+nvm use 14
```

Comment:
```suggestion
nvm use 16
```
14 has dropped off the LTS support list now, so if we are going to recommend something for copy/paste code we should put at least 16 (I think you used 18 fine in the demos?).
https://nodejs.dev/en/about/releases/